examples/basic/README.md

```diff
@@ -1,15 +1,15 @@
 Example: Simple GraphQL Client
 ------------------------
 
-
 Usage:
 
-```
+```bash
 pipenv install -d
 pipenv shell
 export GH_TOKEN=<your github API token>
 
-python examples/basic/http-endpoint.py $GH_TOKEN profusion/sgqlc
+cd examples/basic
+python 01_http_endpoint.py $GH_TOKEN profusion/sgqlc
```

Comment:
would you please use `python3` just to make sure people get it right? not every OS has `python`  symlinked to the `python3` (in all code examples)

Reply:
sure